pages/signup.js

```diff
@@ -30,13 +30,13 @@ export default function Home() {
       <main className={styles.main}>
         <div className={styles["main-container"]}>
           <div className={styles["main-container-content-article-small"]}>
-            <p>CREATE AN ACCOUNT</p>
+            <p>SKAPA ETT KONTO</p>
           </div>
             <SignUp />
         </div>
-        <div className={styles["right-container"]}>
+        {/* <div className={styles["right-container"]}>
           <RightSidebar />
-        </div>
+        </div> */}
```

Comment:
Avoid keeping dead code in the codebase. If it's not going to be used, delete it completely.

Leaving code around creates ergonomics issues for developers working on the code in the future.

With this comment left in the code, if someone wanted to try to comment out the entire `main` by adding a `{/*` snippet above line 30 (e.g. while debugging some issue), that comment would be prematurely terminated here, even though it should have been terminated after line 40 below.